lib/packages/docutils/rstgen_blockdiag.nim

```diff
@@ -0,0 +1,44 @@
+#
+#        Nim's Runtime Library - blockdiag adaptor
+#        (c) Copyright 2017 Federico Ceratto
+#
+#    See the file "copying.txt", included in this
+#    distribution, for details about the copyright.
+#
+#
+## Blockdiag rstgen adaptor
+##
+## See examples/blockdiag.nim for an example.
+
+import math, os, osproc, random
+from rstast import PRstNode
+
+from ../compiler/options import getNimcacheDir
```

Comment:
Ugh, but now docutils depends on the compiler source code. Something we are trying to avoid. Docutils is also used by the forum, at least.